app/tests/evaluation_tests/test_models.py

```diff
@@ -688,6 +688,75 @@ def test_combined_leaderboard_ranks(combined_ranks, expected_ranks):
     assert [cr["rank"] for cr in combined_ranks] == expected_ranks
 
 
+@pytest.mark.parametrize(
+    "eval_status, eval_rank, eval_metrics_json_file_value, expected_missing_metrics",
+    (
+        (Evaluation.FAILURE, 0, {"acc": {"std": 0.1, "mean": 0.0}}, None),
+        (
+            Evaluation.SUCCESS,
+            1,
+            {
+                "acc": {"std": 0.1, "mean": 0.0},
+                "dice": {"std": 0.2, "mean": 0.5},
+            },
+            None,
+        ),
+        (
+            Evaluation.SUCCESS,
+            0,
+            {"acc": {"std": 0.1, "mean": 0.0}},
+            ["dice.mean"],
+        ),
+        (
+            Evaluation.SUCCESS,
+            0,
+            {"dice": {"std": 0.2, "mean": 0.5}},
+            ["acc.mean"],
+        ),
+        (
+            Evaluation.SUCCESS,
+            0,
+            {"cosine": {"std": 0.1, "mean": 0.0}},
+            ["acc.mean", "dice.mean"],
+        ),
+    ),
+)
+@pytest.mark.django_db
+def test_evaluation_missing_metrics(
```

Comment:
I think you are missing the case where the status is `Evaluation.SUCCESS` and the rank is 0 and no metrics are missing.